README.md

```diff
@@ -39,6 +39,20 @@ and cannot guarantee their functionality.
 
 [notesalexp]: https://notesalexp.org/packages/en/source/solvespace/
 
+### Via Snap Store
```

Comment:
Let's put this section above "unofficial binary packages", since these are, in fact, official, and preferred as we control them and can guarantee they're good.

Reply:
Right